### DISCUSSION TITLE
Connecting to the tty of a vm launched in the background.

### DISCUSSION THREAD

**Discussion by rishirsinha** [cloud-hypervisor/cloud-hypervisor#2688] [Q&A]
In the qemu + kvm + libvirt world, I can use ```virsh console``` to connect to the tty console. Is there something in the cloud-hypervisor world to help me connect to the tty console for a VM launched by a hidden cloud-hypervisor process.

**Comment by russell-islam** [comment]
Yes. We have Cloud Hypervisor enabled in libvirt. That allows the above scenario.
https://github.com/cloud-hypervisor/libvirt/tree/ch
